processor/src/stack/mod.rs

```diff
@@ -171,27 +203,13 @@ impl Stack {
             start_pos < MIN_STACK_DEPTH,
             "start position cannot exceed stack top size"
         );
```

Comment:
Could we update the doc comment to this function (above)?

Reply:
fixed